crates/network-scheduler/src/cli.rs

```diff
@@ -45,6 +45,8 @@ pub struct Config {
     pub storage_domain: String,
     pub dataset_buckets: Vec<String>,
     pub scheduler_state_bucket: String,
+    #[serde(skip_serializing)]
+    pub cloudflare_storage_secret: String,
```

Comment:
Because we don't want it to show on [this page](https://scheduler.testnet.subsquid.io/config)
Do you know any other potential ways it (or worker's signature) could leak?